plugin-server/src/utils/db/db.ts

```diff
@@ -412,6 +412,34 @@ export class DB {
     }
 
     // Person
+    REDIS_PERSON_ID_PREFIX = 'person_id'
+    REDIS_PERSON_CREATED_AT_PREFIX = 'person_create_at'
+    REDIS_PERSON_PROPERTIES_PREFIX = 'person_props'
+    REDIS_PERSON_INFO_TTL = 1000
+
+    private async updatePersonIdCache(teamId: number, distinctId: string, personId: number): Promise<void> {
```

Comment:
let's move the `writeToRedis` checks to these functions? That'll make things much cleaner to turn on and off rather than looking for all places where these functions are called.

Reply:
🤦‍♀️  yes, what happens if you try to copy some earlier stuff - the other TEAMS variable that we had in plugin server instead of thinking 😅